src/lib/views/account-settings/SettingsCard.svelte

```diff
@@ -2,31 +2,66 @@
   /* This Source Code Form is subject to the terms of the Mozilla Public
    * License, v. 2.0. If a copy of the MPL was not distributed with this
    * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+  import { createEventDispatcher } from "svelte";
   import Card from "$lib/components/Card.svelte";
   import UpdatePassword from "./UpdatePassword.svelte";
   import UpdateEmail from "./UpdateEmail.svelte";
   import CheckEmailCard from "$lib/components/auth-cards/CheckEmailCard.svelte";
+  import ForgetPwCard from "$lib/components/auth-cards/ForgetPWCard.svelte";
   import DeleteAccount from "./DeleteAccount.svelte";
 
-  export let isEmail;
-  export let isPW;
-  export let isDelete;
-  export let isCheckEmail;
+  const dispatch = createEventDispatcher();
+
   export let cardArgs;
   export let displayCard;
   export let settingsTitle;
   export let settingsDescription;
+  export let isEmail;
+  export let isPW;
+  export let isDelete;
+  export let isCheckEmail;
+  export let isCheckEmailPW;
+  export let isForgetPW;
 
   let checkEmailArgs = {
     ...cardArgs,
     title: "Check your email",
     isSettings: true,
   };
+
+  let checkEmailPWArgs = {
+    ...checkEmailArgs,
+    checkPW: true,
+  };
+
+  let forgetPWArgs = {
+    ...cardArgs,
+    title: "Forgot your password?",
+    isSettings: true,
+    cta1: "Reset password",
+    storyBookTest: false,
+  };
+
+  const handleSelect = (type) => {
+    dispatch("type", {
+      text: type,
+    });
+  };
 </script>
 
-{#if isCheckEmail}
+{#if isCheckEmail && !isCheckEmailPW && !isForgetPW}
```

Comment:
These conditions can be collapsed and simplified as
if (isForgetPW)
-> ForgetCard
else if (isCheckEmail || isCheckEmailPW)
  -> ForgetCard
else
   -> line 65 and below in the updated file.

Reply:
Did you mean !isCheckEmail || !isCheckEmailPW because I don't want to the show the forget pw card if either of those are true.  Or did you mean show the CheckEmail card? There are two different check email cards with different args unless there's another clever way to simplify this?    

Reply:
Would / should forget pw be true if we set any of the other flags to be true (dont we reset?). If so, then we can do:
```
if (isCheckEmail || isCheckEmailPW)
-> ForgetCard
else if (isForgetPW)
-> ForgetCard
else
-> line 65 and below in the updated file.
```